src/activeList.js

```diff
@@ -36,8 +89,9 @@ const tasksListShow = (itemsArr) => {
       parentNode.insertAdjacentHTML('beforeend',
         `<div class="show-list">
         <input type="checkbox" data-done class="check-box">
-      <p class="tast" data-time="${item.index}">${item.description}</p>
+      <p class="tast" data-edit data-time="${item.index}">${item.description}</p>
       <img src="images/drag-drop.png" alt="" class="drag-drop">
+      <img src="images/trash.png" alt="" data-delete class="trash">
       </div>`);
       handleItem(item);
     });
```

Comment:
- [ ] After selecting some task your app keep the good state in the localStorage after a page reload but the UI state is not kept, the selected tasks are become unselected. Let's fix that issue